src/main.tsx

```diff
@@ -56,6 +60,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
                     <Route path="/messages/chat/:peer" element={<ThreadWindow />} />
                     <Route path="/messages/new-chat" element={<NewChat />} />
                     <Route path="/messages/chat-invites" element={<ChatInvites />} />
+                    <Route path="/messages/invite/:account" element={<DirectInvite />} />
```

Comment:
Might just be me but I'm getting a failed to fetch toast on going to this endpoint
<img width="710" alt="image" src="https://user-images.githubusercontent.com/26746725/232718838-d0382714-21fc-4ff4-ad9b-36f2cbc47da4.png">

Reply:
ya it's `/` not `?`

Reply:
right, but this was the url i was going to `https://web3inbox-dev-hidden-git-feat-qrcode-invites-walletconnect1.vercel.app/messages/invite/0x80E50F4655b9B30Cf27D51D59C7a22fE9320eA78`
This screenshot is of my console which was throwing an error for keys.walletconnect.com

Reply:
Was the address registered on the correct keyserver?

Reply:
Yes I just realized, you're not using `CAIP10` address, it should be `eip155:1${address}`

Reply:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/26746725/233052562-f7995412-8af8-438d-9103-59d8e97025e9.png">
Yep that worked perfectly!
Looks sick as well 🔥